templates/upgrade-job.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{.Job.JobName}}
+  namespace: default
+{{if .Labels}}
+  labels:
+    {{range $k,$v := .Labels}}
+    {{$k}}: '{{$v}}'
+    {{end}}
+{{end}}
+spec:
+  template:
+    metadata:
+    {{if .Labels}}
+    labels:
+        {{range $k,$v := .Labels}}
+          {{$k}}: '{{$v}}'
+          {{end}}
+    {{end}}
+    spec:
```

Comment:
I think we need to specify the termination and cleanup as per https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#job-termination-and-cleanup